.pre-commit-config.yaml

```diff
@@ -87,3 +87,5 @@ repos:
           - tenacity
           - types-Pillow
           - pandas-stubs
+          - fastapi
```

Comment:
Can you alphabetize this

Reply:
Weird, I assumed pre-commit would do this for me

Reply:
The only YAML autoformatter we have is `prettier`. I think if we added https://github.com/google/yamlfmt it could work